src/documentation/visualizations/legacy_renderer.malloynb

```diff
@@ -0,0 +1,33 @@
+>>>markdown
+# Legacy Renderer
+
+A new renderer is currently being developed for Malloy; however, some functionality is not yet fully implemented. To revert back to the legacy renderer, use the model tag `## renderer_legacy`.
+
+You will need to revert back to the legacy renderer if you are using:
+- Export to HTML in VS Code
```

Comment:
@skokenes any other features here we should call out?

Reply:
Things not in the new renderer yet off the top of my head:
- drill
- pivot
- transpose